echopype/calibrate/api.py

```diff
@@ -34,11 +36,101 @@ def _compute_cal(
         return cal_obj.compute_Sp(waveform_mode=waveform_mode, encode_mode=encode_mode)
 
 
-def compute_Sv(echodata: EchoData, **kwargs):
-    """Compute volume backscattering strength (Sv) from raw data."""
+def compute_Sv(echodata: EchoData, **kwargs) -> xr.Dataset:
+    """
+    Compute volume backscattering strength (Sv) from raw data.
+
+    The calibration routine varies depending on the sonar type.
+    Currently this operation is supported for the following ``sonar_model``:
+    EK60, AZFP, EK80 (see below for detail).
+
+    The EK80 echosounder can be configured to transmit
+    either broadband (``waveform_mode="BB"``) or narrowband (``waveform_mode="CW"``) signals.
+    When transmitting in broadband mode, the returned echoes are
+    encoded as complex samples (``encode_mode="complex"``).
```

Comment:
So, if waveform_mode="BB", encode_mode is unnecessary b/c compute_Sv can set it to "complex", right?

Reply:
Yes, but I don't remember if we currently allow waveform_mode="BB" and encode_mode=None to work...

Reply:
Hmm, upon looking into this further, it's a bit of a sprawling situation and I need to work on the code itself before updating the docstrings. I put the docstring in `calibrate/api.py` because that's the ones that RTD will see, but the actual kwargs are passed under the hood to the right CALIBRATOR depending on the sonar model. 

The current default and if/else checks are somewhat problematic. As a note for myself, this is where the fixes should go: 
https://github.com/OSOceanAcoustics/echopype/blob/main/echopype/calibrate/calibrate_ek.py#L727

@emiliom I will ping you on this PR again when it is ready for review. I'll also update the PR title to reflect changes made in the code.